Sources/BraintreeDropIn/BTCardFormViewController.m

```diff
@@ -663,8 +666,11 @@ - (void)enrollCard {
                 self.view.userInteractionEnabled = NO;
             });
             
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
             cardRequest.smsCode = authCode;
             cardRequest.enrollmentID = self.enrollmentID;
+#pragma clang diagnostic pop
             
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wdeprecated-declarations"
```

Comment:
Nitpick comment - but we can reduce the number of pragmas since there are some used just below

```suggestion
#pragma clang diagnostic push
#pragma clang diagnostic ignored "-Wdeprecated-declarations"
            cardRequest.smsCode = authCode;
            cardRequest.enrollmentID = self.enrollmentID;

```